<ui version="4.0" >
 <class>SvnRelocateDialog</class>
 <widget class="QDialog" name="SvnRelocateDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Subversion Relocate</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="1" >
      <widget class="QLabel" name="currUrlLabel" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>New repository URL:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="newUrlEdit" >
       <property name="toolTip" >
        <string>Enter the URL of the repository the working space should be relocated to</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Current repository URL:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="insideCheckBox" >
     <property name="toolTip" >
      <string>Select, if the relocate should happen inside the repository</string>
     </property>
     <property name="text" >
      <string>Relocate inside repository (used, if the repository layout has changed)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>newUrlEdit</tabstop>
  <tabstop>insideCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SvnRelocateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>31</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel" >
     <x>29</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SvnRelocateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>69</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel" >
     <x>69</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
